llvm-runtimes/libgcc/libgcc-19.1.4.ebuild

```
@@ -0,0 +1,47 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit crossdev llvm.org toolchain-funcs
+
+DESCRIPTION="Compiler runtime library for clang, compatible with libgcc_s"
+HOMEPAGE="https://llvm.org/"
+
+LICENSE="Apache-2.0-with-LLVM-exceptions || ( UoI-NCSA MIT )"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~mips ~ppc64 ~riscv ~x86 ~amd64-linux ~arm64-macos ~ppc-macos ~x64-macos"
+
+DEPEND="
+	llvm-runtimes/compiler-rt:${LLVM_MAJOR}
+	=llvm-runtimes/libunwind-${PV}
+	!!sys-devel/gcc
+"
+BDEPEND="
+	llvm-core/clang:${LLVM_MAJOR}
+"
+
+LLVM_COMPONENTS=( llvm-libgcc )
+llvm.org_set_globals
+
+src_compile() {
```

```suggestion
src_compile() {
	local -x CC=${CHOST}-clang
```

This doesn't seem to compile with `gcc`.

---

Yeah, that ebuild even conflicts with gcc. Good call, I can overwrite the `CC`. But as said above, for the compatibility with crossdev, I will try `local -x CC=${CTARGET}-clang`.